render_config.sh

```diff
@@ -1,26 +1,22 @@
 ENV=${1:-dev}
-VAULT_TOKEN=${2:-$(cat "$HOME"/.vault-token)}
 
-VAULT_ADDR="https://clotho.broadinstitute.org:8200"
-DRSHUB_VAULT_PATH="secret/dsde/terra/kernel/$ENV/$ENV/drshub"
-COMMON_VAULT_PATH="secret/dsde/terra/kernel/$ENV/common"
-
-VAULT_COMMAND="vault read"
+if [[ $ENV == dev ]]; then
+    GCLOUD_PROJECT=broad-dsde-dev
+elif [[ $ENV == integration ]]; then
+    GCLOUD_PROJECT=broad-dsde-qa
+else
+    echo "Invalid environment: $ENV"
+    exit 1
+fi
 
 SERVICE_OUTPUT_LOCATION="$(dirname "$0")/service/src/main/resources/rendered"
 INTEGRATION_OUTPUT_LOCATION="$(dirname "$0")/integration/src/main/resources/rendered"
 
-if ! [ -x "$(command -v vault)" ]; then
-  VAULT_COMMAND="docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=$VAULT_ADDR vault:1.7.3 $VAULT_COMMAND"
-fi
+GET_SECRET="gcloud secrets versions access latest --project $GCLOUD_PROJECT --secret"
```

Comment:
It's unusual to see `$VARIABLE command` have $VARIABLE itself be the thing executing the command. Perhaps put this in a function?

```
get_secret() {
  gcloud secrets versions access latest --project "$GCLOUD_PROJECT" --secret "$1"
}

get_secret "drshub-ras-mtls-client-cert"
```

Reply:
I was following the existing code pattern here, and `shellcheck` doesn't flag any issues. But I can change it if you like. I read this as `<shell expansion> <stuff after expansion>` not `<command> <argument>` so it seems fine to me.